packages/provider/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@baseapp-frontend/provider",
   "description": "Providers for React Query and Emotion.",
-  "version": "2.0.1",
+  "version": "3.0.0",
```

Comment:
looks like it should've been a patch update instead

Reply:
Updated.